Source/CombatExtended/CombatExtended/Settings.cs

```diff
@@ -75,6 +77,7 @@ public override void ExposeData()
             Scribe_Values.Look(ref allowMeleeHunting, "allowMeleeHunting", false);
             Scribe_Values.Look(ref smokeEffects, "smokeEffects", true);
             Scribe_Values.Look(ref mergeExplosions, "mergeExplosions", true);
+	    Scribe_Values.Look(ref turretsBreakShields, "turretsBreakShields", true);
```

Comment:
and here

Reply:
Not sure how that happened, my editor was set to try to match the whitespace rules of existing files but got itself confused.  I've got it set to force spaces now.  

At some point we should write a `clangformat` spec for the C# and run it on the whole repo, as the style varies a bit from file to file.